cfg/validate.go

```diff
@@ -185,6 +185,13 @@ func isValidMetricsConfig(m *MetricsConfig) error {
 	return nil
 }
 
+func isValidChunkTransferTimeoutForRetriesConfig(chunkTransferTimeoutSecs int64) error {
+	if chunkTransferTimeoutSecs < 0 || chunkTransferTimeoutSecs > maxSupportedTTLInSeconds {
```

Comment:
Why do we need the maxSupportedTTLInSeconds check?
```suggestion
	if chunkTransferTimeoutSecs < 0 {
```

Reply:
maxSupportedTTLInSeconds represents maximum multiple of seconds representable by time.Duration.

To check upper limit.